src/ucp/tag/tag_rndv.c

```diff
@@ -52,6 +108,9 @@ ucs_status_t ucp_tag_rndv_process_rts(ucp_worker_h worker,
         return UCS_OK;
     }
 
+    UCP_WORKER_GET_EP_BY_ID(&ep, worker, rts_hdr->sreq.ep_id, goto cancel,
```

Comment:
when need to do ucp_rndv_unexp_cancel here? the rts just arrived, so it could not already be on unexp queue